### ISSUE TITLE
Firefox: Forking of notebook leads to session reconnect.

### ISSUE THREAD

**Issue by amithrb** [att/rcloud#2062]
Browser: Firefox
Steps to Reproduce:
1. Create a new notebook.
2. Upload two or more assets of size around 100KB each.
3. Now try to fork that notebook, following error is displayed.
![c1](https://cloud.githubusercontent.com/assets/19602717/15962256/566a05a6-2f27-11e6-86e9-b60889a71eb2.PNG)

**Comment by gordonwoodhull** [comment]
Confirmed. This is a weird one. When I tried reloading and forking again, I got an error about "bytecode version is too old" a few times. Inside of `logical(1)` (sorry, can't repro to get the full error trace). 

Also I do not see it on rcloud-5.

So I wonder if there might be an R version problem here.

**Comment by s-u** [comment]
"bytecode version is too old" means indeed that you have stale packages. You must re-install all packages when you change the R version.

**Comment by gordonwoodhull** [comment]
I didn't upgrade my R. Can this result from packages warning that they were compiled for the wrong version of R?

**Comment by gordonwoodhull** [comment]
Nope, I don't think this has to do with package versions. I am able to repro on two OSX instances, and I am not seeing the bytecode error - that was a red herring.

However, I still not have repro'd on any of our Linux servers.
